resources/gin/prepare/tracknet_process_from_txt_to_lists.cfg

```diff
@@ -0,0 +1,14 @@
+#You need to change input file
+process_input_data.file_name='../data/new_data/classifier_1.txt'
+process_input_data.csv_params={
+                    "sep": '\s+',
+                    #"nrows": 1000,
+                    "encoding": 'utf-8',
+                    "names":  ['event',  'x', 'y', 'z', 'station',
+                               'track', 'px', 'py', 'pz', 'X0', 'Y0', 'Z0']
+                 }
+process_input_data.events_quantity = '10'
+process_input_data.radial_stations_constraints = {
```

Comment:
what is this config? Why is uses deprecated params and not uses a list of transforms?